nishimweelysee/Elysee-CapstoneProject
# Ft new article

#### What does this PR do?
This PR is for creating a new article.
>
#### Description of Task to be completed
>- Here there are articles like blog or a portfolio
>- There create blog tab and create portfolio tab
>- there is an input for title
>- image input
>- introduction input
>- article description
>- input for a description
>- a link for an article
>- Save button to create an article
>
#### How should this be manually tested?
> Open a website and login as admin 
> go to admin page
> go blog or portfolio tab 
> you will see a form to fill in order to save your article.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
>FT(Create new Article) Use this link  https://trello.com/c/4lViJyxt to visit my trello board.
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/92328447-2915a600-f061-11ea-9c52-86db8da50471.png)
![image](https://user-images.githubusercontent.com/56052588/92328454-3af74900-f061-11ea-8f40-ab4ef1a458f1.png)

Those above are for BLog articles.

![image](https://user-images.githubusercontent.com/56052588/92328509-988b9580-f061-11ea-8dd4-ea0d88eafdce.png)

![image](https://user-images.githubusercontent.com/56052588/92328515-a4775780-f061-11ea-9079-31ae7aab26c6.png)

Those are for Portfolio

Thank you!!
